crates/language/src/target_language.rs

```diff
@@ -204,6 +204,7 @@ impl PatternLanguage {
         match name {
             "js" => match flavor {
                 Some("jsx") => Some(Self::Tsx),
+                Some("flow") => Some(Self::Tsx),
```

Comment:
We should also support `flowComments` or remove it from the grammar: https://github.com/getgrit/tree-sitter-gritql/blob/5f4543610f49f7607b5ca54082da017837219a23/grammar.js#L64C1-L65C1